### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 3034saurabhkumar**
I made some changes like grammatical mistakes, article mistakes and few spelling mistakes to make it easier to understand your project.

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
It makes it easier for a new contributor to an understanding of the project.

#### Changes proposed in this pull request:

- Spelling mistakes
- Grammatical mistakes 

Fixes #

### CODE REVIEW

**Comment by iamareebjamal** [comment]
I see no grammatical correction. Most are subjective changes that don't warrant a PR

**Comment by 3034saurabhkumar** [comment]
> I see no grammatical correction. Most are subjective changes that don't warrant a PR



> I see no grammatical correction. Most are subjective changes that don't warrant a PR

please see the changes in line number 254 and 299

File: README.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ We use **Sendgrid** to send mails, and you need a `Sendgrid API` to make it work
 
 ### Github Deploy
 
-The web-app is able to automatically deploy the generated event sites to Github Pages of a User. We use `passport` library for authentication of the user. For uploading and committing the event files, we use `github` library which provides a wrapper for the **Github API** in `node`.
+The web-app can automatically deploy the generated event sites to Github Pages of a User. We use the `passport` library for authentication of the user. For uploading and committing the event files, we use the `Github` library which provides a wrapper for the **Github API** in `node`.
```
Comment (iamareebjamal) [inline README.md:254]: I see no grammatical fix here

File: README.md [hunk 2]
```diff
@@ -294,13 +294,13 @@ The web-app is able to automatically deploy the generated event sites to Github
 
 > Before making a pull request, please file an issue. So, other developers have the chance to give feedback or discuss details. Match every pull request with an issue please and add the issue number in description e.g. like "**Fixes #123**".
 
-**We have the following branches :-**
+**We have the following branches:-**
  * **development**
-	 All development goes on in this branch. If you're making a contribution,
+	 All development goes on in this branch. If you're contributing,
```
Comment (iamareebjamal) [inline README.md:299]: What's the grammatical correction here? You just changed the way of saying the same thing, previous version is a completely valid english sentence

**Pull request closed without merging**